components/renku_data_services/utils/core.py

```diff
@@ -0,0 +1,9 @@
+"""Shared utility functions."""
+import ssl
+
+
+def get_ssl_context():
```

Comment:
Also lets toss in a functools.lru_cache on the `get_ssl_context` function. Because we really dont need this to execute every single time. I just worry about it because now we will open, read and parse a file before every http request sent with httpx.